core/src/main/java/org/apache/iceberg/rest/HTTPClient.java

```diff
@@ -448,6 +443,43 @@ static HttpRequestInterceptor loadInterceptorDynamically(
     return instance;
   }
 
+  static HttpClientConnectionManager getConnectionManager(
```

Comment:
the codebase avoids using `get` as a prefix, so I think `configureConnectionManager` might make more sense here

Reply:
Thanks for catching this. Just saw this in the [coding guidelines](https://iceberg.apache.org/contribute/#method-naming) as well.